client/app/domain/dns/domain-dns.state.js

```diff
@@ -0,0 +1,14 @@
+const state = {
+  url: '/dns',
+  views: {
+    domainView: {
+      templateUrl: 'domain/dns/DNS.html',
```

Comment:
```javascript
import mytemplate from "foo/bar.html";
// ...
domainView: {
  template: mytemplate,
}
```
that way webpack will report errors if template is missing

Reply:
```javascript
import template from "foo/bar.html";
// ...
domainView: {
  template,
}
```